.ci/azure/linux.yml

```diff
@@ -81,10 +81,10 @@ jobs:
       unzip gradle-$(GRADLE_VER)-bin.zip
       # For opencv-python: python3-setuptools and pip upgrade
       python3 -m pip install --upgrade pip
-      python3 -m pip install -r $(OPENVINO_REPO_DIR)/inference-engine/ie_bridges/python/requirements.txt
+      python3 -m pip install -r $(OPENVINO_REPO_DIR)/src/bindings/python/requirements.txt
       python3 -m pip install -r $(OPENVINO_REPO_DIR)/src/bindings/python/wheel/requirements-dev.txt
       # For running Python API tests
-      python3 -m pip install -r $(OPENVINO_REPO_DIR)/inference-engine/ie_bridges/python/src/requirements-dev.txt
+      python3 -m pip install -r $(OPENVINO_REPO_DIR)/src/bindings/python/src/requirements-dev.txt
```

Comment:
Let's install https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/requirements_test.txt instead of -dev

Reply:
done)

Reply:
Thanks! However seems like we need a different file. See comment below